deployment/docker-compose.yml

```diff
@@ -7,162 +7,93 @@
 #
 # See accompanying Make commands for easy collectstatic etc.
 
-smtp:
-  # Note you cannot scale if you use container_name
-  container_name: bims-smtp
-  image: catatnight/postfix
-  hostname: postfix
-  environment:
-    # You could change this to something more suitable
-    - maildomain=kartoza.com
-    - smtp_user=noreply:docker
-  restart: unless-stopped
+version: '2'
+services:
+  smtp:
+    image: catatnight/postfix
+    hostname: postfix
+    environment:
+      # You could change this to something more suitable
+      - maildomain=kartoza.com
+      - smtp_user=noreply:docker
+    restart: unless-stopped
 
-db:
-  # Note you cannot scale if you use container_name
-  container_name: bims-db
-  image: kartoza/postgis:9.6-2.4
-  volumes:
-    #- ./pg/postgres_data:/var/lib/postgresql
-    - ./backups:/backups
-    - ./sql:/sql
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-  restart: unless-stopped
-#  Uncomment the next line to have an access with PGAdmin using localhost and port 25432 on your computer.
-#  Only for development !
-#  ports:
-#    - "25432:5432"
+  db:
+    image: kartoza/postgis:9.6-2.4
+    environment:
+      - ALLOW_IP_RANGE=0.0.0.0/0
```

Comment:
I forget my own configs, but doesn't this open 5432 to all internet? Probably not what we want.... For Limpopo we'll need to work with the client o find out the specific address range that should be given access...

Reply:
That problem is not relevant with local dev environment IMO. We want to open it to all possible docker subnetwork so it will work in mac as nicely in linux. On prod environment, we use catalog for rancher and the subnet is secured by rancher even though we allow all range connection. We could use restricted IP address if we want, but I don't know how we define it on rancher.

Reply:
ok thanks